src/story.test.ts

```diff
@@ -58,7 +58,7 @@ CSF2Story.loaders = [() => Promise.resolve({ d: '3' })];
 CSF2Story.args = { a: 1 };
 
 const CSF3Story: XStory = {
-  render: (args) => 'Named Story',
+  render: args => 'Named Story',
```

Comment:
Your prettier settings are off. This should have parenthesis.

Reply:
IDK, I think this must be a setting on the repo? 
![image](https://user-images.githubusercontent.com/132554/177921721-2d442219-3c4b-41fa-a521-a4c473a97c21.png)

Reply:
![image](https://user-images.githubusercontent.com/132554/177921906-2ff4ab64-25e2-4d9f-aa38-1504445336ca.png)

Reply:
Ok we should fix that to align with Storybook I suppose.